src/onnx/parse_dequantizelinear.cpp

```diff
@@ -21,40 +20,19 @@ struct parse_dequantizelinear : op_parser<parse_dequantizelinear>
         if(contains(info.attributes, "axis"))
             axis = info.attributes.at("axis").i();
 
-        auto input_lens = args[0]->get_shape().lens();
-        int n_dim       = static_cast<int>(input_lens.size());
-
-        auto sub_zero_point = args[0];
+        auto x       = args[0];
+        auto x_scale = args[1];
 
         if(args.size() == 3)
```

Comment:
We do not need this check here. The operator can have two or three inputs. So we can just do:

```
return info.add_instruction(make_op("dequantizelinear", {{"axis", axis}}), args)
```

Reply:
The same for the quantizelinear operator

Reply:
I had it this way so that rewrite_(de)quantizelinear.cpp could assume that there's always a third argument. Better to keep it the way it is, or make your suggested change and add the check to the rewrite...cpp files?